templates/helpers/gettodaysdate.js

```diff
@@ -0,0 +1,19 @@
+module.exports = () => {
```

Comment:
want an _easy_ way of writing this?
try:
```js
var options = { day: 'numeric', month: 'long', year: 'numeric' };
return new Date().toLocaleDateString('en-GB', options);
```
see: https://repl.it/Cbfv/2 (_output is identical_) 😉